easybuild/easyconfigs/g/GIBBON/GIBBON-20250127.eb

```
@@ -0,0 +1,39 @@
+easyblock = 'Tarball'
+
+name = 'GIBBON'
+version = '20250127'
+_commit = '6da45f4'
+
+homepage = 'https://www.gibboncode.org/'
+description = """
+GIBBON (The Geometry and Image-Based Bioengineering add-On) is an open-source
+MATLAB toolbox by Kevin M. Moerman and includes an array of image and geometry
+visualization and processing tools and is interfaced with free open source
+software such as TetGen, for robust tetrahedral meshing, and FEBio for finite
+element analysis. The combination provides a highly flexible image-based
+modelling environment and enables advanced inverse finite element analysis."""
+
+toolchain = SYSTEM
+
+sources = [{
+    'source_urls': ['https://github.com/gibbonCode/GIBBON/archive'],
+    'download_filename': f'{_commit}.tar.gz',
+    'filename': SOURCE_TAR_GZ,
+}]
+patches = ['%(name)s-%(version)s_fix.patch']
+checksums = [
+    {'GIBBON-20250127.tar.gz': 'b8efa94f2d304f2a1a90cb35812a3dde7a99d4586f1d30c466205823e29d6533'},
+    {'GIBBON-20250127_fix.patch': 'ee9d0f983cd91f7f4a8125d4887b454a87b1f1d046892abea82f854e7292e7d0'},
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib', 'lib_ext', 'docs'],
+}
+
+_mpaths = ['.', 'lib', 'lib_ext', 'docs'] + [
+    f'lib_ext/{x}' for x in ['export_fig', 'fcw', 'findjobj', 'geogram', 'gmsh', 'tetGen', 'xml_io_tools']]
+
+modextrapaths = {'MATLABPATH': _mpaths}
+
```

I would make more clear that this needs MATLAB:
```suggestion

modloadmsg = '%(name)s requires MATLAB to work. You can load any available version of MATLAB.'

```